sys/mathoid.yaml

```diff
@@ -0,0 +1,86 @@
+info: 
```

Comment:
This file should not be used at all, all of Mathoid's logic is handled in the public spec-module.